skopt/tests/test_optimizer.py

```diff
@@ -105,3 +105,16 @@ def test_acq_optimizer(base_estimator):
         opt = Optimizer([(-2.0, 2.0)], base_estimator=base_estimator,
                         n_random_starts=1, acq_optimizer='lbfgs')
     assert 'The tree-based regressor' in str(e.value)
+
+
+def test_exhaust_random_calls():
+    # check a model is fitted if there is at least one call to tell
+    base_estimator = ExtraTreesRegressor(random_state=2)
+    opt = Optimizer([(-2.0, 2.0)], base_estimator, n_random_starts=2,
+                    acq_optimizer="sampling")
+
+    x = opt.ask()  # random point
+    opt.tell(x, 3.)
+    opt.ask()  # random point
```

Comment:
Is this is a smoke test? I would prefer to test two things.

1. Repeated calls to ask give different points.
2. `self.models` etc is empty till `n_points` are told.